enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metabase/embedding-sdk-react",
-  "version": "0.1.0",
+  "version": "0.1.2",
```

Comment:
This version has already been published. I believed this should be bumped every time re published a new version.